rpcs3/Emu/Cell/Modules/cellMic.h

```diff
@@ -345,6 +346,7 @@ class microphone_device
 
 	std::vector<mic_device> devices;
 	std::vector<u8> temp_buf;
+	std::vector<u8> float_buf;
```

Comment:
I think you also have to increase the size of:
`simple_ringbuf<inbuf_size> rbuf_dsp;`
below to:
`simple_ringbuf<inbuf_size * 2> rbuf_dsp;`

Or we may run into the ensure in write_bytes